### PR TITLE
Changelog for node release 8.10

### PR DESCRIPTION
**Pull request by teodanciu**
# Description

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
      [versioning process](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process).
- [ ] The version bounds in `.cabal` files for all affected packages are updated. **If you change the bounds in a cabal file, that package itself must have a version increase.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process))
- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages. **New section is never added with the code changes.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd))
- [x] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
- [x] Cabal files are formatted (use `scripts/cabal-format.sh`)
- [x] [`hie.yaml`](https://github.com/intersectmbo/cardano-ledger/blob/master/hie.yaml) has been updated (use `scripts/gen-hie.sh`)
- [ ] Self-reviewed the diff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,29 @@ the `CHANGELOG.md` for each individual package for any changes relevant for deve
...
 
+## 8.10
+
+- Retention of Anchor for proposal procedures in the ledger state in order for users to be able to query the node for those anchors
+- The minimum fee calculation has been updated to take into consideration a new protocol parameter controlling the contribution of reference scripts sizes
+- Redeemers serialization improvement: encode redeemers as a Map, instead of a list for Conway
+- Support for intra-era hardfork initiation and necessary updates to the SPO stake distribution calculation.
+- Fail translation when Conway features are present in transactions that use Plutus v1/v2
+- New Conway predicate failure: `ConwayUtxosPredFailure` (to decouple Conway from `PPUP` dependency)
+- Remove `small-steps-test` package and convert it into a sublibrary of `smalls-steps`
```
Comment (lehins) [inline CHANGELOG.md]:
```suggestion
- Remove `small-steps-test` package and convert it into a sublibrary of `small-steps`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,29 @@ the `CHANGELOG.md` for each individual package for any changes relevant for deve
...
+- Support for injecting transition configuration data in `NewEpochState`, to decouple this concern from Consensus
+- Query-related changes:
+  - Era-specific queries: Conway functionality supported only starting with Conway era
+  - Change committee query to return non-optional `CommitteeMembersState`
+  - Add optional `Anchor`to resigned status in `CommitteeMembersState`
```
Comment (lehins) [inline CHANGELOG.md]:
```suggestion
  - Add optional `Anchor` to resigned status in `CommitteeMembersState`
```

**Review by lehins — APPROVED** [review]
Looks awesome. Thank you for writing it up!

File: CHANGELOG.md (hunk 1 above)
Comment (teodanciu) [reply CHANGELOG.md]: Fixed, thank you!

**Pull request merged**